lib/ruby-debug-ide/xml_printer.rb

```diff
@@ -6,10 +6,24 @@
 module Debugger
 
   class MemoryLimitError < StandardError  
+    attr_reader :message
+    attr_reader :backtrace
+
+    def initialize(message, backtrace)
```

Comment:
This one does not provide default `backtrace` value and the one below does. Is that intentional?